Shared/ViewModels/DevicesViewModel.swift

```diff
@@ -0,0 +1,251 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Combine
+import Foundation
+import JellyfinAPI
+import OrderedCollections
+import SwiftUI
+
+final class DevicesViewModel: ViewModel, Stateful {
+
+    // MARK: - Action
+
+    enum Action: Equatable {
+        case getDevices(_ userID: String?)
```

Comment:
Instead, this class should take the user ID in its `init` so that it's always associated with it.